src/components/paniniCreator/formSections/TextSection.tsx

```diff
@@ -1,18 +1,24 @@
+import { useFormContext } from "react-hook-form";
 import styles from "./FormSection.module.css";
 import { FormSectionProps } from "./FormSectionProps";
 import FormSectionTemplate from "./FormSectionTemplate";
-
 interface TextSectionProps extends Omit<FormSectionProps, "options" | "removable"> {
   removable?: never;
```

Comment:
**nitpick:** Why never? I think it should be `boolean`

Reply:
Because on figma this form section didn't have options to delete/add i've assumed that text input doesn't need such feature, hence i didn't implement it, and made necessary changes to type so it wouldn't accept removals (options)